build.sbt

```diff
@@ -49,6 +49,7 @@ lazy val commonSettings = Seq(
   assemblyJarName in assembly := "scalafmt.jar",
   ScoverageSbtPlugin.ScoverageKeys.coverageExcludedPackages :=
       ".*Debug;.*ScalaFmtLogger;.*Versions",
+  cleanFiles <+= baseDirectory(_ / "repos.tar.gz"),
```

Comment:
Nope, but it's a PR open for discussion :) Easier to add a line and submit PR than to ask. Will remove.